### PR TITLE
Add tooltips to the data labels for Search Rate graph

### PR DESCRIPTION
**Pull request by Afani97**
Ticket:
- https://app.clickup.com/t/863gea6f1

Changes:
- Add a util function to return correct copy for each stop purpose
- Add a popperjs tooltip to render above each stop purpose when hovering
- Add convenient make command to refresh stop summary 

Preview:
Screen Shot 2023-05-02 at 5 55 20 PM

### CODE REVIEW

File: frontend/src/Components/Charts/SearchRate/SearchRate.js [hunk 1]
```diff
@@ -170,24 +198,50 @@ function SearchRate(props) {
...
-                toolTipFontSize={7}
-              />
+              <>
+                <Tooltip
+                  ref={setPopperElement}
+                  style={{
+                    ...styles.popper,
+                    maxWidth: '500px',
+                    zIndex: 1000,
+                    display: tooltipText !== '' ? 'block' : 'none',
```
Comment (Afani97) [inline frontend/src/Components/Charts/SearchRate/SearchRate.js:208]: This smoothes the show/hide animation a bit, looked choppy without it.

File: frontend/src/Components/Charts/SearchRate/SearchRate.js [hunk 2]
```diff
@@ -170,24 +198,50 @@ function SearchRate(props) {
...
+                  data={chartData}
+                  loading={chartState.loading[LIKELIHOOD_OF_SEARCH]}
+                  horizontal
+                  iAxisProps={{
+                    tickLabelComponent: (
+                      <VictoryLabel
+                        x={100}
+                        dx={-50}
+                        style={{ fontSize: 6 }}
+                        id={(t) => (Array.isArray(t.text) ? t.text.join('') : null)}
```
Comment (Afani97) [inline frontend/src/Components/Charts/SearchRate/SearchRate.js:225]: Set the id which we reference to show the different tooltip language above.

File: frontend/src/Components/Charts/SearchRate/SearchRate.js [hunk 3]
```diff
@@ -52,6 +54,32 @@ function SearchRate(props) {
...
+        options: {
+          offset: [0, 10],
+        },
+      },
+    ],
+  });
+
+  const showTooltip = (e) => {
+    setReferenceElement(e.currentTarget);
+    setTooltipText(tooltipLanguage(e.target.parentElement.id));
```
Comment (Afani97) [inline frontend/src/Components/Charts/SearchRate/SearchRate.js:74]: Reason we need `parentElement` is the graphing library breaks up each label by word into span tags. The parent elem's id has the reference to the to stop purpose the user hovers over to then update the tooltip text.

File: frontend/src/util/tooltipLanguage.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+export const tooltipLanguage = (stopCause) =>
+  ({
+    Average: 'REPLACE ME',
```
Comment (Afani97) [inline frontend/src/util/tooltipLanguage.js:3]: Waiting to hear about this language that wasn't in the basecamp post: https://3.basecamp.com/3120230/buckets/18119898/messages/3864796186

File: Makefile [hunk 5]
```diff
@@ -33,3 +33,9 @@ dl-db-dump:
 	export AWS_PROFILE=copwatch
 	inv aws.configure-eks-kubeconfig
 	inv staging pod.get-db-dump --db-var=DATABASE_URL_NC
+
+refresh-view:
+	@echo 'Migrating all databases'
+	./migrate_all_dbs.sh
+	@echo 'Refreshing the StopSummary materialized view'
+	python manage.py refresh_pgviews --database=traffic_stops_nc
```
Comment (Afani97) [inline Makefile:41]: Can't ever remember this command, adding here for exposure

File: frontend/src/util/tooltipLanguage.js (hunk 4 above)
Comment (copelco) [reply frontend/src/util/tooltipLanguage.js:3]: I'd also be okay setting this to `''` and adding average later

**Review by copelco — APPROVED** [review]

**Pull request merged**